src/get-started/java-binding.md

```diff
@@ -362,6 +412,25 @@ service:
 - Follow the instructions in the [application guide][app-tutorial] to configure
   and start an Exonum node with your service.
 
+## Common Library
+
+Java Binding includes a library module that can be useful for Java client
+applications that interact with an Exonum service and
+does not have the dependency on Java Binding Core. The module contains Java
```

Comment:
Java Binding includes a library module that can be useful for Java client
applications that interact with an Exonum service. The module does not have the dependency on Java Binding Core, but it contains Java classes obligatory for core that can now as well be easily applied in clients, if necessary.